sklearn/gaussian_process/kernels.py

```diff
@@ -685,10 +685,25 @@ def requires_vector_input(self):
 
 
 class Sum(KernelOperator):
-    """Sum-kernel k1 + k2 of two kernels k1 and k2.
+    """The `Sum` kernel takes two kernels :math:`k_1` and :math:`k_2`
+    and combines them via
 
-    The resulting kernel is defined as
-    k_sum(X, Y) = k1(X, Y) + k2(X, Y)
+    .. math::
+        k_{sum}(X, Y) = k_1(X, Y) + k_2(X, Y)
+
+    Using the Sum-kernel is equivalent to adding two kernels::
+
+
+        RBF() + RBF()
+
+    is the same as::
+
+        Sum(RBF(), RBF())
```

Comment:
I'd leave the previous docstring, TBF. We usually try to avoid latex in docstrings for readability of the code, but most importantly this is redundant with the UG and it increases the risk of inconsistencies. So it makes maintenance a bit harder.

The link to the UG is very welcome though.

Reply:
Since I added latex to most kernel docstrings, would you prefer to have them replaced with plain text in all of them as before?
I personally find formulas in plain text or code rather difficult to read, especially more complex ones as in some of the kernels and prefer latex (but having a math background, I acknowledge I'm biased here).
The previous plain text formulas in the docstring actually had the problem that they had inconsistencies with the UG, so I copy-pasted formulas from there for consistency.

The other option would be to remove formulas completely from the docstrings, refer to the UG and only leave explanations of the parameters in plain text.

Reply:
hmm, I was very happy with the latex ones though. It makes the docs very nicely readable IMO, and I have gotten the feedback from users that they are confused when there's no explanation in the API doc. So I'll be happy to leave them here.

Reply:
Honestly I personally don't mind latex in docstrings but last time I tried, it got rejected (by @agramfort IIRC, pinging in case you want to comment) 

If we keep the latex, then please remove the rest "    Using the Sum-kernel..." which is now redundant.

Reply:
haha, didn't know that. But looking at the codebase, we already do have some latex, and they look pretty good to me :D

Reply:
I don't see it as redundant since I wanted to highlight that instead of the function one could also use the + operator but agree that the formulation is maybe not the best.
Would this be better?
```
Using the Sum-kernel is equivalent to using the + operator::

        RBF() + RBF()

    is the same as::

        Sum(RBF(), RBF())
```

Reply:
:wave: @adrinjalali any chance to merge this PR soon? :sweat_smile:

Reply:
@corriebar this already has my +1, waiting for @NicolasHug to finish his review :)